services/src/test/java/io/scalecube/services/ServiceConfigTest.java

```diff
@@ -13,17 +13,22 @@
   @Test
   public void test_services_config_port_increment_false() {
 
-    Microservices node = Microservices.builder().clusterConfig(
-        ClusterConfig.builder()
-
+    Microservices node = Microservices.builder()
+        .serviceTransport(TransportConfig.builder()
+            .port(3000)
+            .portAutoIncrement(false)
+            .build())
+        .clusterConfig(ClusterConfig.builder()
             .transportConfig(TransportConfig.builder()
-                .port(3000)
+                .port(4000)
```

Comment:
just fyi, actually `ClusterConfig` has shortcut methods for `port` and `portAutoIncrement ` so you can call thm directly there